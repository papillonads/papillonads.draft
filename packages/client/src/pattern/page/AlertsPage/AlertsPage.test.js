import React from 'react'
import alertsPageDefault from './AlertsPage.render'

jest.mock('react-apollo', () => {
  return {
    Query: () => <div />,
  }
})

describe('<AlertsPage />', () => {
  describe('Snaphot', () => {
    test('must match alertsPageDefault', () => {
      expect(global.renderToJSON(alertsPageDefault)).toMatchSnapshot()
    })
  })
})
