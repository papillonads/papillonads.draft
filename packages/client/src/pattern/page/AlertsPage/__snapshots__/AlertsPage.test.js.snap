// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AlertsPage /> Snaphot must match alertsPageDefault 1`] = `<div />`;
